fossology/fossology
# chore(ununpack): Remove external files

## Description

Remove external files from ununpack agent which are used to calculate the checksum of files.

Also, remove OpenSSL dependency from delagent due to recent changes in OpenSSL license.

See #1462 for more info.

### Changes

1. Remove external C and H files from ununpack agent.
1. Use [libgcrypt](https://dev.gnupg.org/source/libgcrypt/) instead (LGPL-2.1).
1. Remove OpenSSL dependency from delagent, use `libgcrypt`.

## How to test

1. Check if ununpack and wgetagent works during file upload.
1. Check if delagent works (both from CLI and UI).

Closes #1462
